### DISCUSSION TITLE
Error downloading formula for CERN's root

### DISCUSSION THREAD

**Discussion by cp-laserstorm** [Homebrew/discussions#346] [Casks]
Hey all,

I'm currently trying to download the formula for CERN's ROOT library on my Mac (macOS Catalina, version 10.15.6). According to CERN's site ([https://root.cern/install/#macos-package-managers](url)) the homebrew command is "brew install root". However, typing that gets me the error "root: no bottle available!" Not entirely sure what I've done wrong here, as I'm new to homebrew. If anyone knows what could be the problem, I'd appreciate the help!

**Comment by carlocab** [comment]
What does `brew config` and `brew doctor` say?

**Reply by cp-laserstorm** [reply]
Writing `brew config` gets me this:
`HOMEBREW_VERSION: 2.7.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: 993bf95877143afe1cede022083499e0b4fe336c
Last commit: 11 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a74d940b9fb6df03303bfe2379e79a7d85f04a32
Core tap last commit: 69 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.CCiEgUyX0d/org.macosforge.xquartz:0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 11.0 build 1103
Git: 2.24.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.6-x86_64
CLT: 11.5.0.0.1.1588476445
Xcode: N/A
XQuartz: 2.7.11 => /opt/X11`

And writing `brew doctor` gets me this:
`Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you an update run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.


Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /opt/anaconda3/bin/icu-config
  /opt/anaconda3/bin/krb5-config
  /opt/anaconda3/bin/freetype-config
  /opt/anaconda3/bin/xslt-config
  /opt/anaconda3/bin/libpng16-config
  /opt/anaconda3/bin/libpng-config
  /opt/anaconda3/bin/xml2-config
  /opt/anaconda3/bin/python3-config
  /opt/anaconda3/bin/curl-config
  /opt/anaconda3/bin/ncursesw6-config
  /opt/anaconda3/bin/pcre-config
  /opt/anaconda3/bin/python3.8-config
  /Library/Frameworks/Python.framework/Versions/2.7/bin/python2-config
  /Library/Frameworks/Python.framework/Versions/2.7/bin/python2.7-config
  /Library/Frameworks/Python.framework/Versions/2.7/bin/python-config`

**Reply by carlocab** [reply]
I think your issue is that you don't have Xcode installed. Please install it from the App Store then try again. Upgrading your version of the Command Line Tools may help too. See the output of `brew doctor` for how to do this.

If that doesn't work, please paste the complete output of `brew install root` here.

**Reply by cp-laserstorm** [reply]
Yeah, I don't have XCode, so that would make sense.

I'm a big fan of VSCode though, if I downloaded this through XCode could I get the package to run on VSCode?

**Reply by carlocab** [reply]
I believe you should be able to; the Xcode requirement should just be for dependencies in the Xcode toolchain. However, I don't use `root` so I can't say for sure.
